fe/fe-core/src/test/java/com/starrocks/lake/StarOSAgentTest.java

```diff
@@ -673,4 +675,32 @@ public List<WorkerGroupDetailInfo> listWorkerGroup(String serviceId, List<Long>
         Assert.assertEquals("127.0.0.1:8090", addresses.get(0));
         Assert.assertEquals("127.0.0.2:8091", addresses.get(1));
     }
+
+    @Test
+    public void testUpdateShardGroup() throws StarClientException, DdlException, UserException {
```

Comment:
this case is just for code coverage, no valuable test point at all.

Reply:
OK I'll remove it later.